app/assets/javascripts/components/usage/Stats_Display.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+
+function StatsDisplay() {
+  const StatsURL = 'http://localhost:3000/usage.json';
+  const [stats, setStats] = useState([]);
+  const fetchStats = () => {
+    fetch(StatsURL)
+    .then(res => res.json())
+    .then(data => setStats(data.stats));
+  };
+  useEffect(() => {
+    fetchStats();
+  }, []);
```

Comment:
You're fetching the data twice - once here and once in the `Usage` component. Better to pass the stats object as a prop to this component.